jobsdb/jobsdb.go

```diff
@@ -460,6 +461,21 @@ func (jd *HandleT) Status() interface{} {
 	if db.IsValidMigrationMode(jd.migrationState.migrationMode) {
 		statusObj["migration-state"] = jd.migrationState
 	}
+	pendingEventMetrics := metric.GetManager().GetRegistry(metric.PUBLISHED_METRICS).GetMetricsByName(fmt.Sprintf(metric.JOBSDB_PENDING_EVENTS_COUNT, jd.tablePrefix))
```

Comment:
[minor] avoiding long lines:

```suggestion
	pendingEventMetrics := metric.GetManager()
	 .GetRegistry(metric.PUBLISHED_METRICS)
	 .GetMetricsByName(fmt.Sprintf(metric.JOBSDB_PENDING_EVENTS_COUNT, jd.tablePrefix))
```